### DISCUSSION TITLE
Style icons from module

### DISCUSSION THREAD

**Discussion by JohnKesko** [ng-icons/ng-icons#70] [General]
First of all, great work! :)

My question is pretty simple I guess. Can we style the icons from a module?
I managed to import all iconoir and set the size but can we also set the style?

import { NgModule } from '@angular/core';
import { BrowserModule } from '@angular/platform-browser';
import { AppComponent } from './app.component';
import { NgIconComponent, provideIcons, provideNgIconsConfig } from '@ng-icons/core';
import * as ionicons from '@ng-icons/iconoir';

@NgModule({
  declarations: [
    AppComponent,
  ],
  imports: [
    BrowserModule,
    NgIconComponent
  ],
  providers: [
    provideIcons(ionicons),
    provideNgIconsConfig({
      size: '1.4em',
     color: 'black' <---- something like this?
    }),
  ],
  bootstrap: [AppComponent]
})
export class AppModule { }

**Comment by ashley-hunter** [comment]
Hi, thank you, and thanks for the suggestion, I have just released an update with global color functionality added, if you have any issues let me know, thanks!
